src/vs/editor/contrib/snippet/snippetVariables.ts

```diff
@@ -49,6 +50,7 @@ export const KnownSnippetVariableNames: { [key: string]: true } = Object.freeze(
 	'WORKSPACE_FOLDER': true,
 	'RANDOM': true,
 	'RANDOM_HEX': true,
+	'RANDOM_UUID': true
```

Comment:
Since they are always random a better name might be just `UUID`

Reply:
updated